specification/maps/data-plane/DataRegistry/stable/2022-12-01/dataregistry.json

```diff
@@ -0,0 +1,712 @@
+    },
+    {
+      "SasToken": []
+    }
+  ],
+  "responses": {},
+  "parameters": {
+    "ApiVersion": {
+      "name": "api-version",
+      "description": "Version number of Azure Maps API.",
+      "type": "string",
+      "in": "query",
+      "required": true,
+      "default": "2022-12-01",
+      "x-ms-parameter-location": "client"
+    },
+    "DataFormat": {
+      "name": "dataFormat",
+      "description": "Data format of the content being uploaded.",
+      "type": "string",
+      "in": "query",
+      "required": true,
+      "enum": [
+        "geojson",
+        "zip"
+      ],
+      "x-ms-enum": {
+        "name": "DataFormat",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "geojson",
+            "description": "[GeoJSON](https://tools.ietf.org/html/rfc7946) is a JSON based geospatial data interchange format."
+          },
+          {
+            "value": "zip",
+            "description": "Compressed data format."
+          }
+        ]
+      },
+      "x-ms-parameter-location": "method"
+    },
+    "UniqueDataId": {
+      "name": "udid",
+      "description": "The unique data id for the data registry. The `udid` must have been obtained from a successful [Data Upload API](https://docs.microsoft.com/en-us/rest/api/maps/data-v2/upload-preview) call.",
+      "type": "string",
+      "in": "path",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "DataDescription": {
+      "name": "description",
+      "description": "The description to be given to the data registry.",
+      "type": "string",
+      "in": "query",
+      "x-ms-parameter-location": "method"
+    },
+    "OperationId": {
+      "name": "operationId",
+      "type": "string",
+      "in": "path",
+      "description": "The ID to query the status for the data upload request.",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    }
+  },
+  "paths": {
+    "/mapData": {
+      "post": {
+        "description": "***Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\nThe DataRegistry Create API allows the caller to create a data registry for their data content uploaded to Azure Storage Account in the Azure Maps Service and in exchange of a UDID. You can use this API in a scenario like registering a collection of Geofences in `GeoJSON` format, for use in our [Azure Maps Geofencing Service](https://docs.microsoft.com/rest/api/maps/spatial).\n\n## Submit Create Request\n\nTo create a data registry you will use a `POST` request. The request body will contain the connection object. The `dataFormat` query parameter will contain the format for the data for validation.\n\nFor example, to upload a collection of geofences  in `GeoJSON` format, set the request body to the connection object. Set the `dataFormat` query parameter to _geojson_.\n\nThe DataRegistry Create API performs a [long-running request](https://aka.ms/am-creator-lrt-v2).",
```

Comment:
> The DataRegistry Create API allows the caller to create a data registry
The term, **data registry**, is quite clear to developers like us which means a record in our data registry table.
But I think it's not that clear from customer perspective.
Maybe use something like "The DataRegistry Create API allows the caller to register their data to DataRegistry".